src/onegov/election_day/assets/js/jquery.tablesorter.js

```diff
@@ -639,7 +639,7 @@
 			}
 			c.$table.find( c.selectorHeaders ).attr({
 				scope: 'col',
-				role : 'columnheader'
```

Comment:
This is actually an asset from https://github.com/Mottie/tablesorter, if we update this in the future, the change will be lost. If this change is reasonable in general, you should open a pull request there (and keep this change). If not, we should try to introduce this change in an asset fully controlled by us or alternatively at least document this somewhere, in a comment in `get_common_asset`.

Reply:
I created a PR for tablesorter.

Reply:
There hasn't been an update of tablesorter since 2020, I will try the second option of making the change in an asset.